contracts/contracts/token/TrustlinesNetworkToken.sol

```diff
@@ -63,11 +75,25 @@ contract TrustlinesNetworkToken {
         return _totalSupply;
     }
 
+    /**
+    * @dev Transfer token for a specified address
+    * @param to The address to transfer to.
+    * @param value The amount to be transferred.
+    */
     function transfer(address recipient, uint256 amount) public returns (bool) {
         _transfer(msg.sender, recipient, amount);
         return true;
     }
 
+    /**
```

Comment:
The whole point of this change was to specify the behavior of calling approve with MAX_INT so please add this.